measurements/openapi/measurements.yml

```diff
@@ -0,0 +1,424 @@
+
+  /measurements:
+    get:
+      summary: ''
+      description: ''
+      operationId: 'measurements.api.list_measurements'
+      parameters:
+        - name: report_id
+          in: query
+          type: string
+          description: The report_id to search measurements for
+        - name: input
+          in: query
+          type: string
+          description: The input (for example a URL or IP address) to search measurements for
+        - name: probe_cc
+          in: query
+          type: string
+          description: The two letter country code
+        - name: probe_asn
+          in: query
+          type: string
+          description: the Autonomous system number in the format "ASXXX"
+        - name: test_name
+          in: query
+          type: string
+          description: The name of the test
+        - name: since
+          in: query
+          type: string
+          description: >-
+            The start date of when measurements were run (ex.
+            "2016-10-20T10:30:00")
+        - name: until
+          in: query
+          type: string
+          description: >-
+            The end date of when measurement were run (ex.
+            "2016-10-20T10:30:00")
+        - name: since_index
+          in: query
+          type: string
+          description: Return results only strictly greater than the provided index
+        - name: order_by
+          in: query
+          type: string
+          description: 'By which key the results should be ordered by (default: `test_start_time`)'
+          enum:
+            - report_id
+            - input
+            - test_start_time
+            - probe_cc
+            - probe_asn
+            - test_name
+            # These are all equivalent
+            - index
+            - idx
+            - report_no
+        - name: order
+          in: query
+          type: string
+          description: |-
+            If the order should be ascending or descending (one of: `asc` or `desc`)
+          enum:
+            - asc
+            - desc
+            - ASC
+            - DESC
+        - name: offset
+          in: query
+          type: integer
+          description: 'Offset into the result set (default: 0)'
+        - name: limit
+          in: query
+          type: integer
+          description: 'Number of records to return (default: 100)'
+      responses:
+        '200':
+          description: Returns the list of measurement IDs for the specified criteria
+          schema:
+            $ref: "#/definitions/MeasurementList"
+
+        default:
+          description: Default response
+
+  /version:
+    get:
+      summary: ''
+      description: ''
+      operationId: 'measurements.api.get_version'
+      responses:
+        '200':
+          description: Returns version information
+          schema:
+            $ref: "#/definitions/VersionInfo"
+
+
+        default:
+          description: Default response
+
+definitions:
+  VersionInfo:
```

Comment:
So, this is basically the thing you refer to when you do:

> $ref: "#/definitions/VersionInfo"

?

Reply:
Correct.